### PR TITLE
New package: PulseBroswer.PulseBrowser version 1.0.0-a.87

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136775)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 674.)

File: manifests/p/PulseBroswer/PulseBrowser/1.0.0-a.87/PulseBroswer.PulseBrowser.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: PulseBroswer.PulseBrowser
+PackageVersion: 1.0.0-a.87
+InstallerType: burn
+InstallerSwitches:
+  Custom: /S
+UpgradeBehavior: install
+ReleaseDate: 2023-12-01
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/p/PulseBroswer/PulseBrowser/1.0.0-a.87/PulseBroswer.PulseBrowser.installer.yaml:12]:
```suggestion
- Architecture: x64
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 679.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-30T19:26:30.1534872Z ##[error] One or more errors occurred. (Manifest is invalid.) Manifest is invalid. (Line: 11, Col: 17, Idx: 320) - (Line: 11, Col: 17, Idx: 320): While scanning a plain scalar value, found invalid mapping.

(Automated response - build 695.)

File: manifests/p/PulseBroswer/PulseBrowser/1.0.0-a.87/PulseBroswer.PulseBrowser.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+PackageIdentifier: PulseBroswer.PulseBrowser
+PackageVersion: 1.0.0-a.87
+InstallerType: burn
+InstallerSwitches:
+  Custom: /S
+UpgradeBehavior: install
+ReleaseDate: 2023-12-01
+    Dependencies:
+      PackageDependencies:
+        - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (stephengillie) [inline manifests/p/PulseBroswer/PulseBrowser/1.0.0-a.87/PulseBroswer.PulseBrowser.installer.yaml:13]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

I'm not sure where the extra spaces came from.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**